kml/edit-kml-ground-overlay/README.md

```diff
@@ -35,4 +36,4 @@ The [image](https://libapps.s3.amazonaws.com/accounts/55937/images/1944.jpg) is
 
```

Comment:
In about the data, it's possible our alt text could do with updating: was this included in Trevor's updates he did in the common samples? If not might be worth checking with him if there's any room for improvement here.

Reply:
Good catch. I've had a go at updating following the pattern used in the other samples. Might have to change slightly if we get a different feedback.